foundation/foundation-configuration-store/pom.xml

```diff
@@ -23,8 +23,9 @@
                     <version>${version.failsafe.plugin}</version>
                     <configuration>
                         <systemPropertyVariables>
-                            <KRAUSENING_BASE>./target/test-classes/configurations/base</KRAUSENING_BASE>
-                            <KRAUSENING_EXTENSIONS>./target/test-classes/configurations/example-env</KRAUSENING_EXTENSIONS>
+                            <BASE_PROPERTY_URI>./target/test-classes/configurations/base</BASE_PROPERTY_URI>
```

Comment:
A: Since we're moving to PVC based approach, we should remove/replace the URI bit of the name so we don't mislead people.